app/scripts/dimApp.i18n.js

```diff
@@ -112,34 +209,89 @@
           Intellect: "Intellekt",
           Discipline: "Disziplin",
           Strength: "Stärke",
-          Super_cooldown: "Super Abklingzeit",
-          Grenade_cooldown: "Granaten Abklingzeit",
-          Melee_cooldown: "Nahkampf Abklingzeit",
-          tier_progress: "{{progress}} für {{tier}}",
+          SuperCooldown: "Super Abklingzeit",
+          GrenadeCooldown: "Granaten Abklingzeit",
+          MeleeCooldown: "Nahkampf Abklingzeit",
+          TierProgress: "{{progress}} für {{tier}}",
           About: "Über",
-          'Support DIM': "DIM Unterstützen",
+          SupportDIM: "DIM Unterstützen",
           Vendors: "Händler",
           Support: "Unterstützen",
           Settings: "Einstellungen",
-          'Tag Item': "Tag Artikel",
+          FilterHelp: "Suche nach Gegenstand/Perk oder is:arc",
+          TagItem: "Gegenstand taggen",
```

Comment:
@crulplay @Korben85 Would `Ding` or `Stück` be understandable in this context? `Gegenstand` is a rather long word to fit in these locations.